doc/rose-rug-suites-I.html

```diff
@@ -702,35 +708,34 @@ <h3 class="alwayshidden">Directives</h3>
     [[hello_eris]]
 ...
         [[[job submission]]]
-            method = loadleveler
+            method = pbs
         [[[directives]]]
-            class            = serial
-            job_type         = serial
-            notification     = error
-            resources        = ConsumableMemory(5000)
-            wall_clock_limit = 1260,1200
+            -l select = 1
+            -l walltime = 00:05:00
+            -q = normal
 </pre>
     </div>
 
     <div class="slide">
       <h3 class="alwayshidden">Job Submission Methods</h3>
 
-      <p>cylc supports the following job submission methods (potentially more
-      later):</p>
+      <p>cylc supports the following job submission methods (and more):</p>
 
       <ul class="incremental">
         <li>at</li>
 
         <li>background (default)</li>
 
-        <li>loadleveler</li>
+        <li>LoadLeveler</li>
```

Comment:
please change the cases of these to reflect how they would be used a.la. http://cylc.github.io/cylc/html/single/cug-html.html#11.2 (slurm is the incorrect cased entry in this list rather than the others needing camel casing).